conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -26,9 +26,12 @@ show_channel_urls: true
 CONDARC
 )
 
+HOST_USER_ID=$( hash docker-machine 2>/dev/null && docker-machine ssh $(docker-machine active) 'id -u $(whoami)' || id -u $(whoami) )
```

Comment:
Thanks @gqmelo - I appreciate you taking the time. I am convinced by your suggestion. As long as a *feedstock* (not a recipe) conceptually has the ability to control whether it builds the recipe as root or any another user, I'm 👍 .